roles/disaster_recovery/README.md

```diff
@@ -0,0 +1,93 @@
+| dr_source_map           | primary               | Specify the default source map to be used in the play.<br/> The source map indicates the key which is used to get the target value for each attribute which we want to register with the VM/Template.       |
+| dr_reset_mac_pool       | True                  | If True, then once a VM will be registered, it will automatically reset the mac pool, if configured in the VM.        |
+| dr_cleanup_retries_maintenance       | 3                  | Specify the number of retries of moving a storage domain to maintenance VM as part of a fail back scenario.       |
+| dr_cleanup_delay_maintenance       | 120                  | Specify the number of seconds between each retry as part of a fail back scenario.       |
+| dr_clean_orphaned_vms        | True                  | Specify whether to remove any VMs which have no disks from the setup as part of cleanup.       |
+| dr_clean_orphaned_disks        | True                  | Specify whether to remove lun disks from the setup as part of engine setup.       |
+| dr_running_vms		 | /tmp/ovirt_dr_running_vm_list	 | Specify the file path which is used to contain the data of the running VMs in the secondary setup before the failback process run on the primary setup after the secondary site cleanup was finished. Note that the /tmp folder is being used as default so the file will not be available after system reboot.
+
+
+
+Dependencies
+------------
+
+No.
+
+Example Playbook
+----------------
+
+```yaml
+---
+- name: Setup oVirt environment
+  hosts: localhost
+  connection: local
+  vars_files:
+     - ovirt_passwords.yml
+     - disaster_recovery_vars.yml
+  roles:
+     - {role: oVirt.disaster-recovery}
+```
+
+Generate var file mapping [demo](https://youtu.be/s1-Hq_Mk1w8)
+<br/>
+Fail over scenario [demo](https://youtu.be/mEOgH-Tk09c)
+
+Scripts
+-------
+The ovirt-dr script should provide the user a more convenient way to run
+disaster recovery actions as a way to avoid using ansible playbooks directly.
+There are four actions which the user can execute:
+- `generate`	Generate the mapping var file based on the primary and secondary setup, to be used for failover and failback
+- `validate`	Validate the var file mapping which is used for failover and failback
+- `failover`	Start a failover process to the target setup
+- `failback`	Start a failback process from the target setup to the source setup
+
+Each of those actions are using a configuration file whose default location is `oVirt.disaster-recovery/files/dr.conf`<br/>
+The configuration file's location can be changed using `--conf-file` flag in the `ovirt-dr` script.<br/>
+Log file and log level can be configured as well through the `ovirt-dr` script using the flags `--log-file` and `--log-level`
+
+
+Example Script
+--------------
+For mapping file generation:
+```console
+$ ./ovirt-dr generate --log-file=ovirt-dr.log --log-level=DEBUG
```

Comment:
Probably change the path of the ovirt-dr

Reply:
Why should I change it?
This is an example. In this example the log file `ovirt-dr.log` will be created in the current folder. This parameter may be omitted at all, in that case the created log file will be located under `/tmp/`.
Or maybe I didn't understand what you suggested :)

Reply:
Previously the `ovirt-dr` was on top of the project. Now the script is under `roles/disaster_recovery/` so might be better to change the path to something like`./roles/disaster_recovery/ovirt-dr` just suggestion. I'm open for any dissgusion

Reply:
Done.
Handled in the commit that handles all the `README.md` changes.